client/js/app.js

```diff
@@ -119,14 +120,22 @@ const init = () => {
     backToTop(backToTopEl, 'back-to-top--visible');
   }
 
-  nodeList(galleries).forEach((gallery) => {
+  nodeList(articleGalleries).forEach((gallery) => {
     contentSlider(gallery, {
       slideSelector: '.image-gallery__item',
       modifiers: ['in-article', 'in-content', 'gallery'],
       sliderType: 'gallery'
     });
   });
 
+  nodeList(basicPageGalleries).forEach((gallery) => {
+    contentSlider(gallery, {
+      slideSelector: '.image-gallery__item',
+      modifiers: ['in-basic', 'in-content', 'gallery'],
+      sliderType: 'gallery'
+    });
+  });
```

Comment:
:nitpicking: could we not just have `.js-image-gallery` and `.js-image-gallery-basic` instead of using the top class to be more specific?